### PR TITLE
Design and HTML markup for fellowship page

### PR DESCRIPTION
**Pull request by olasitarska**
I used the styling from [overview page](https://www.djangoproject.com/start/overview/) for this. Not sure about the last part (What's Fellowship and Who are Fellows), maybe it should be the same width as the rest? It feels quite good if you look at the website, looks weird on the screenshot though. 

I recommend clicking on the screenshot below and zooming in to 100% for a good experience ;)

screencapture-127-0-0-1-8000-fundraising

Suggestions welcome :) 

cc @asendecka @bmispelon @jezdez @treyhunner

### CODE REVIEW

**Comment by asendecka** [comment]
Great work! :+1: 

I suggest putting heart higher, before the text about supporting the program. It would be awesome to see immediately the heart and see call for action on top of the page.

**Comment by asendecka** [comment]
I also think it's a good idea to hide old "Heros": display 4 weeks only and the rest as a toggle.

**Comment by olasitarska** [comment]
@asendecka :heart: 

- Good point with the heart, I think swapping two sections (intro and heart) would be definitely better :) 
- I thought about only showing 2 weeks, notice the pagination after second week -- I thought we could just paginate them there?:)

File: djangoproject/templates/fundraising/index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  </div>
+
+</div>
+
+  <div class="section">
+
+  </div>
+  {% endblock %}
+
+  {% block content-related %}
```
Comment (timgraham) [inline djangoproject/templates/fundraising/index.html]: remove this block? Not sure it'll ever be relevant to this page and if so, I'd expect it'd be an included template.

Comment (olasitarska) [inline djangoproject/templates/fundraising/index.html]: Definitely, not sure how I missed that! :)

File: djangoproject/templates/fundraising/index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+  {% endblock %}
+
+  {% block content-extra %}
+
+  <div class="layout-tertiary" id="about">
+    <div class="container">
+        <h2>What is Django Fellowship?</h2>
+        <p><a href="https://www.djangoproject.com/weblog/2014/sep/18/dsf-calls-applicants-django-fellow/">Django Fellowship program</a> is a project where paid contractors have been engaged to manage some of the administrative and community management tasks of Django project. The initial pilot took three months and came to close in mid-January; one of the major goals is to secure funding for this program on an ongoing basis.</p>
+        <p>The first three months of the Django Fellowship program have been very successful. In this short time Django Fellows worked on around 240 pull requests (including revising some of the major features that will be in Django 1.8), triaged at least 90 different tickets and made around 200 commits in total!</p>
```
Comment (timgraham) [inline djangoproject/templates/fundraising/index.html]: It's much easier to review the text if it isn't on one long line. I can do some copyediting on the text -- do you want me to add it to your branch or merge this first and then create a separate PR?

Comment (olasitarska) [inline djangoproject/templates/fundraising/index.html]: Let's focus on design / html markup here, there is #292 for all copywriting issues (which I'm sure there are :)). Make sense?

Comment (timgraham) [inline djangoproject/templates/fundraising/index.html]: Sure, overall it looks like a fine starting point then.

**Comment by olasitarska** [comment]
Ok, I'll add some design for testimonial when I get home (in an hour?) and this will be ready to be merged :)

**Comment by olasitarska** [comment]
Final design:

screencapture-127-0-0-1-8000-fundraising 1

**Pull request merged**